ts/framework.d.ts

```diff
@@ -1,122 +1,122 @@
 declare module DevExpress.framework {
-    /** @docid_ignore dxCommandContainer */
-    /** @docid_ignore dxCommandContaineroptions_id */
-    /** @docid_ignore dxcontent */
-    /** @docid_ignore dxcontentoptions_targetPlaceholder */
-    /** @docid_ignore dxcontentplaceholderoptions_contentCssPosition */
-    /** @docid_ignore dxcontentplaceholderoptions_name */
-    /** @docid_ignore dxcontentplaceholderoptions_transition */
-    /** @docid_ignore dxcontentplaceholderoptions_animation */
-    /** @docid_ignore dxcontentplaceholder */
-    /** @docid_ignore dxtransitionoptions_name */
-    /** @docid_ignore dxtransitionoptions_type */
-    /** @docid_ignore dxtransitionoptions_animation */
-    /** @docid_ignore dxtransition */
-    /** @docid_ignore dxlayoutoptions_name */
-    /** @docid_ignore dxlayout */
-    /** @docid_ignore dxviewoptions_disableCache */
-    /** @docid_ignore dxviewoptions_modal */
-    /** @docid_ignore dxviewoptions_name */
-    /** @docid_ignore dxviewoptions_orientation */
-    /** @docid_ignore dxviewoptions_pane */
-    /** @docid_ignore dxviewoptions_title */
-    /** @docid_ignore dxview */
-    /** @docid_ignore dxviewPlaceholderoptions_viewName */
-    /** @docid_ignore dxviewPlaceholder */
-
-    /** @docid ViewCache */
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+    
+
+    /** An object that stores information about views displayed in the application. */
     export class ViewCache {
 
-        /** @docid ViewCacheevents_viewRemoved */
+        
```

Comment:
Where is a description for this member?
See: https://js.devexpress.com/Documentation/ApiReference/SPA_Framework/ViewCache/Events/#viewRemoved